File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.js
function solution(rsp) {
    var answer = '';
    for (let r of rsp) {
        switch(r) {
            case '2':
                answer += 0;
                break;
            case '0':
                answer += 5;
                break;
            case '5':
                answer += 2;
                break;
        }
    }
    return answer;
}

File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.java
class Solution {
    public String solution(String rsp) {
        StringBuilder answer = new StringBuilder("");
        for (char r : rsp.toCharArray()) {
            switch(r) {
                case '2':
                    answer.append('0');
                    break;
                case '0':
                    answer.append('5');
                    break;
                case '5':
                    answer.append('2');
                    break;
            }
        }
        return answer.toString();
    }
}